scripts/broad_dsde_workflows/HaplotypeCallerGvcfScatterWf_170204.wdl

```diff
@@ -0,0 +1,147 @@
+## Copyright Broad Institute, 2017
+## 
+## This WDL workflow runs HaplotypeCaller in GVCF mode on a single sample, scattered 
+## across intervals.
+##
+## Requirements/expectations :
+## - One analysis-ready BAM file for a single sample (as identified in RG:SM)
+## - Set of variant calling intervals lists for the scatter, provided in a file
+##
+## Outputs :
+## - One GVCF file and its index
+##
+## Cromwell version support 
+## - Successfully tested on v24
+## - Does not work on versions < v23 due to output syntax
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation. 
+## For program versions, see docker containers. 
+##
+## LICENSING : 
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in 
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may 
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker 
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+# TASK DEFINITIONS
+
+# HaplotypeCaller per-sample in GVCF mode
```

Comment:
Eh, that's kind of a slippery slope... By the same logic we could link every single tool call to docs. I don't have the stomach to take that on right now (though for the future it would be nice).